pkg/http/server.go

```diff
@@ -335,8 +335,13 @@ func (s *server) unescapeRequestParametersHandler(next fasthttp.RequestHandler)
 func (s *server) getRouter(endpoints []Endpoint) *routing.Router {
 	router := routing.New()
 	parameterFinder, _ := regexp.Compile("/{.*}")
+
+	// Build the API allowlist and denylist
+	allowedAPIs := s.apiSpec.Allowed.ForProtocol("http")
```

Comment:
So it is better if we make all support protocols as const.

Reply:
Done